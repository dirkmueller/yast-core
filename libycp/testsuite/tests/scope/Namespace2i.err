[Scanner] tests/scope/Namespace2i.ycp:15 Error: Bad or unknown file 'Namespace2'
runycp: parser error
runycp: parser error
[Scanner] tests/scope/Namespace2i.ycp:16 Error: Unknown identifier 'Namespace2'
[Scanner] tests/scope/Namespace2i.ycp:16 Error: parse error, unexpected $end
runycp: parser error
[Scanner] tests/scope/Namespace2i.ycp:16 Error: parse error, unexpected ')'
runycp: parser error
runycp: parser error
Parsed:
----------------------------------------------------------------------
4711
----------------------------------------------------------------------
runycp: parser error
runycp: parser error
runycp: parser error
[Scanner] tests/scope/Namespace2i.ycp:20 Error: Bad or unknown file 'Namespace2'
runycp: parser error
runycp: parser error
[Scanner] tests/scope/Namespace2i.ycp:21 Error: Unknown identifier 'Namespace2'
[Scanner] tests/scope/Namespace2i.ycp:21 Error: parse error, unexpected $end
runycp: parser error
[Scanner] tests/scope/Namespace2i.ycp:21 Error: parse error, unexpected ')'
runycp: parser error
runycp: parser error
Parsed:
----------------------------------------------------------------------
nil
----------------------------------------------------------------------
runycp: parser error
runycp: parser error
runycp: parser error
[Scanner] tests/scope/Namespace2i.ycp:25 Error: Bad or unknown file 'Namespace2'
runycp: parser error
runycp: parser error
[Scanner] tests/scope/Namespace2i.ycp:26 Error: Unknown identifier 'Namespace2'
[Scanner] tests/scope/Namespace2i.ycp:26 Error: parse error, unexpected $end
runycp: parser error
[Scanner] tests/scope/Namespace2i.ycp:26 Error: parse error, unexpected ')'
runycp: parser error
runycp: parser error
Parsed:
----------------------------------------------------------------------
4753
----------------------------------------------------------------------
runycp: parser error
runycp: parser error
runycp: parser error
