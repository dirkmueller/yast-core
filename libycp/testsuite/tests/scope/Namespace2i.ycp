
# ---------------------------------------------------------
#
#  Filename:    Namespace2.ycp
#
#  Purpose:     test named scopes
#
#  Creator:     mvidner@suse.cz
#
#  Maintainer:  mvidner@suse.cz
#
# ---------------------------------------------------------

{
   import "Namespace2";
   return (Namespace2::globalFunction() == 4711);
}

{
   import "Namespace2";
   return (Namespace2::localFunction() == nil);
}

{
   import "Namespace2";
   return (Namespace2::mixedFunction() == 4753);
}
