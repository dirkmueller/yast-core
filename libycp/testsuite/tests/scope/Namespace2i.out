(4711)
(nil)
(4753)
